cert-manager/smaug/issuer.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: ingress-letsencrypt-production
+  namespace: openshift-ingress
+spec:
+  acme:
+    email: ops-team@operate-first.cloud
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: ingress-certificate
+    solvers:
+      - selector:
+          dnsZones:
+            - "na.operate-first.cloud"
+      - dns01:
+          cnameStrategy: Follow
+          cloudDNS:
+            project: aicoe-prow
+            hostedZoneName: 'na-operate-first-cloud'
+            serviceAccountSecretRef:
+              key: sa
+              name: gcp-sa
```

Comment:
can you rename this to `clouddns-dns01-solver-svc-acct` I like this name more and it matches the one [here](https://cert-manager.io/docs/configuration/acme/dns01/google/#set-up-a-service-account)